src/QtUtils/include/QtUtils/FutureWatcher.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/types/span.h>
+
+#include <QObject>
+#include <chrono>
+
+#include "OrbitBase/Future.h"
+
+namespace orbit_qt_utils {
+
+/*
```

Comment:
nit: maybe use // comment style everywhere? I think we use it by default

Reply:
Good catch, I wasn't aware of that. Fixed.